ZIP 0 updates (#674)
zcash/zips: zip-0000.rst

you can always submit a competing ZIP.

If you are interested in assuming ownership of a ZIP, send a message
asking to take over, addressed to all of the current Owner(s) and the
ZIP Editors. If the Owner(s) who are proposed to be removed don't respond
in a timely manner, the ZIP Editors and any remaining current Owners will

> make a decision (it's not like such decisions can't be reversed :).

Reviewer: ```suggestion
make a decision (such decisions may be reversed).
```
